Database/Updates/Shard/01-06-16-2017-SetCharacterDeletedFlag.sql

```diff
@@ -0,0 +1,18 @@
+DELIMITER $$
+
+CREATE DEFINER=`root`@`localhost` EVENT `SetCharacterDeletedFlag` ON SCHEDULE EVERY 15 MINUTE STARTS '2017-06-16 16:15:35' ON COMPLETION NOT PRESERVE ENABLE DO BEGIN
```

Comment:
Can change to `EVERY 1 MINUTE` and the net effect is the character is deleted from the character list on time instead of ~15 minutes after deleteTime